### PR TITLE
Remove min-width

### LINKED ISSUE #1741: Datepicker (inputfield): minimal width question
Datepicker's README.md correctly states that the minimal width is 197px:

>### width
>String-valued `width` allows to override the intrinsic dimensions of the datepicker. Default is `100%`.
>Note that there is a minimum of `260px` if you use it inline and `197px` if you use it as a inputfield.

image

The reason for this minimal width is missing....
It is then tempting to wish a smaller width, so that space could be spared in situations like this:

image

Please advice, thanks! 

(came up in discussion with Jonas on 21.4.2020)

### PR DESCRIPTION
**Pull request by danyball**
Fixes #1741 

# Done is when (DoD):
- [x] Modifications within components are reflected by tests.
- [x] A self-review of the code has been performed.
- [ ] The modified component properties have been added to the typescript typings.
- [x] Potential design changes have been reviewed by a designer.
- [x] The modifications are well documented (README.md, etc.) and showcased in the stories.
- [x] The modifications are documented in the code, particularly in hard-to-understand areas (focus on **WHY**).
- [x] The modifications are shortly added to CHANGELOG.md with the issue number.
- [x] The modifications still work in IE.
- [x] The modifications are according to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)

### CODE REVIEW

**Review by raphaellueckl — DISMISSED** [review]
lgtm

File: src/components/20-molecules/datepicker/ui.test.js [hunk 1]
```diff
@@ -687,16 +708,17 @@ fixture('Datepicker as inputfield with 196px width and 10px height').page(
...
+test('should have no minimum width', async t => {
   const datepicker = await Selector(() =>
     document.querySelector('axa-datepicker')
   );
   const datepickerInputWrap = await datepicker.find(
     '.m-datepicker__input-wrap'
   );
   await t.expect(datepicker.clientWidth).eql(196); // component has no min-width
-  await t.expect(datepickerInputWrap.clientWidth).eql(197); // input wrapper has min-width
+  await t.expect(datepickerInputWrap.clientWidth).eql(196); // input wrapper has no min-width
```
Comment (01231) [inline src/components/20-molecules/datepicker/ui.test.js:719]: What is the reason for this 1px difference? (197 -> 196)

**Review by 01231 — DISMISSED** [review]

File: src/components/20-molecules/datepicker/index.scss [hunk 2]
```diff
@@ -30,9 +29,10 @@ axa-datepicker {
     }
   }
 
-  &[error]:not([error=""]),
+  &[error]:not([error='']),
```
Comment (LucaMele) [inline src/components/20-molecules/datepicker/index.scss]: Why change to single quotes? I don’t know if this works in all browsers. Standard is “”. I would revert

File: src/components/20-molecules/datepicker/ui.test.js (hunk 1 above)
Comment (danyball) [reply src/components/20-molecules/datepicker/ui.test.js:719]: The min-width. It was 197px. And now without that css rule we can have 196 (or lower) px width.

File: src/components/20-molecules/datepicker/index.scss (hunk 2 above)
Comment (danyball) [reply src/components/20-molecules/datepicker/index.scss]: Prettier is fixing that...?! Will check my prettier rules.

Comment (danyball) [reply src/components/20-molecules/datepicker/index.scss]: IntelliJ is not using .prettierignore if i start prettier manuelly. Will change back to `"`

**Review by markus-walther — APPROVED** [review]
LGTM. I trust you tested it well for visual breakage....

File: src/components/20-molecules/datepicker/index.scss (hunk 2 above)
Comment (markus-walther) [reply src/components/20-molecules/datepicker/index.scss]: I agree with Luca, make it double quotes.

**Comment by danyball** [comment]
> LGTM. I trust you tested it well for visual breakage....

Jep. You can play around on storybook feature branch and set its width at the knobs.

**Pull request merged**